jax/_src/scipy/stats/betabinom.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+
+import scipy.stats as osp_stats
+
+from jax import lax
+from jax._src.numpy.util import _wraps
+from jax._src.numpy.lax_numpy import _promote_args_inexact, _constant_like, where, inf, logical_or
+from jax._src.scipy.special import betaln
+
+
+@_wraps(osp_stats.betabinom.logpmf, update_doc=False)
+def logpmf(k, n, a, b, loc=0):
+    k, n, a, b, loc = _promote_args_inexact("betabinom.logpmf", k, n, a, b, loc)
+    y = lax.sub(lax.floor(k), loc)
+    one = _constant_like(y, 1)
+    zero = _constant_like(y, 0)
+
+    
+    combiln = lax.neg(
+        lax.add(
+            lax.log(lax.add(n, one)),
```

Comment:
Should `lax.log(lax.add(n, one))` be replaced with `lax.log1p(n)`?

Reply:
Yes!  Still learning some of the `lax` functions, so if you see more of that let me know.